notebooks/stage/watson_keywords.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<p>This reminds me that we should update our license...</p><p> </p><p>https://github.com/aicoe-aiops/mailing-list-analysis-toolkit/blob/master/LICENSE</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>